src/Service/Context/PageContext.php

```diff
@@ -67,7 +74,6 @@ class PageContext {
     'account_name' => 'account_id',
```

Comment:
Cool, since we already have consensus. [Done](https://github.com/acquia/acquia_lift/pull/232/commits/95f07ccabc825597cc4a99143fd0e6e3b0e7f200).